www_src/blocks/generator.js

```diff
@@ -11,18 +12,19 @@ Generator.IMAGE = "image";
 
 Generator.generateBlock = function(options) {
   if (options.type === Generator.TEXT) {
-    return <Text value={options.value} />;
+    return <Text {...options} />;
   }
   if (options.type === Generator.LINK) {
-   return <Link href={options.href} label={options.label} active={options.active} />;
+   return <Link {...options} />;
   }
   if (options.type === Generator.IMAGE) {
-    return <Image src={options.src} alt={options.alt} />;
+    return <Image {...options} />;
```

Comment:
Does ES6 +JSX give us some way to compact this to something like `<${type} {...options}/>`? =/